sdk/wallet-adapter/wallet-kit-core/src/index.ts

```diff
@@ -241,26 +241,34 @@ export function createWalletKitCore({
     },
 
     signTransaction(transaction) {
```

Comment:
Lets also update these methods to mirror the adapters.

Reply:
Do we really want to do this now? Currently we expose to dapps the account address only, if we change the type to SuiSignTransactionInput it will require exposing accounts as WalletAccount/ReadOnlyWalletAccount to dapps in order to be able to provide the account param.